Tests/installation_tests/cocoapods/with_frameworks_objc/CocoapodsTest/ViewController.m

```diff
@@ -0,0 +1,30 @@
+//
+//  ViewController.m
+//  CocoapodsTest
+//
+//  Created by Ben Guo on 3/20/18.
+//  Copyright © 2018 Stripe. All rights reserved.
+//
+
+#import "ViewController.h"
+#import <Stripe/Stripe.h>
```

Comment:
I wasn't sure whether we wanted to use `#import` or `@import` - they both seemed to reproduce the warning. Is there a good reason to prefer one or the other here?

Reply:
I think they're effectively identical – `#import` maps to `@import` if modules are enabled, which is on by default in new projects.

https://stackoverflow.com/a/18947634